src/ResourceManager/Resources/Commands.Resources/Models.ResourceGroups/ResourceClient.cs

```diff
@@ -157,22 +157,21 @@ internal List<PSPermission> GetResourcePermissions(ResourceIdentifier identity)
             var permissionsResult = AuthorizationManagementClient.Permissions.ListForResource(
                     identity.ResourceGroupName,
 #if !NETSTANDARD
-                    resourceIdentity);
+					identity.ResourceProviderNamespace,
+					identity.ParentResource,
+					identity.ResourceType,
+					identity.ResourceName);
 #else
                     resourceIdentity.ResourceProviderNamespace,
                     resourceIdentity.ParentResourcePath??"",
                     resourceIdentity.ResourceType,
                     resourceIdentity.ResourceName);
 #endif
 
-            if (permissionsResult != null)
-            {
-                return permissionsResult
-#if !NETSTANDARD
-                    .Permissions
```

Comment:
Yes, because when we use '!NETSTANDARD' Nuget generated from azure-sdk-for-net, we don't need to use '.Permissions', So I deleted it.

'